documentation/source/simulator_support.rst

```diff
@@ -15,6 +17,34 @@ Accessing bits of a vector doesn't work:
 
 See ``access_single_bit`` test in :file:`examples/functionality/tests/test_discovery.py`.
 
+Wavefoms
+~~~~~~~~
+
+To get waveform in vcd format some verilog code must be added in the top
+component as example below:
+
+.. code-block:: verilog
+
+    module button_deb(
+        // sync design
+        input    clk,
+        input    rst,
+        // in-out
+        input    button_in,
+    output button_valid);
```

Comment:
Please fix the indentation, and you can remove the comments on the ports.

Reply:
done